cmd/createProjects.go

```diff
@@ -26,11 +26,12 @@ var createProjectCmd = &cobra.Command{
 func init() {
 	createCmd.AddCommand(createProjectCmd)
 
+	createProjectCmd.Flags().String("project-name", "", "name of the project")
 	createProjectCmd.Flags().Bool("force-create", false, "ignore if the URL is used by another Kondukto project")
 	createProjectCmd.Flags().StringP("overwrite", "w", "", "rename the project name when creating a new project")
 	createProjectCmd.Flags().StringP("labels", "l", "", "comma separated label names")
 	createProjectCmd.Flags().StringP("team", "t", "", "project team name")
-	createProjectCmd.Flags().String("repo-id", "r", "URL or ID of ALM repository")
+	createProjectCmd.Flags().String("repo-id", "", "URL or ID of ALM repository")
```

Comment:
The same"project-create" commands exist on the "scan" command.
You should add the same commands there. Please check the "scan" command.

Reply:
👍